src/Loader.php

```diff
@@ -595,6 +595,8 @@ public static function add_component_settings( $settings ) {
 		// @todo On merge, once plugin images are added to core WooCommerce, `wcAdminAssetUrl` can be retired,
 		// and `wcAssetUrl` can be used in its place throughout the codebase.
 		$settings['wcAdminAssetUrl'] = plugins_url( 'images/', dirname( __DIR__ ) . '/woocommerce-admin.php' );
+		$settings['wcVersion'] = WC_VERSION;
```

Comment:
Minor, but these = could be indented. (Not sure about you, but my vendor scripts like phpcs are also not working right now).